elastic/beats
# [Heartbeat] Add stringer redaction to synthexec cmd

## Proposed commit message

Redact synthetics cmd output to prevent logging of user-defined parameters.

## Checklist




## Disruptive User Impact

N/A

## How to test this PR locally

1. Build heartbeat locally.
2. Add a browser monitor with any playwrightOpts or params.
3. Run and check logs show playwrightOpts or params as `REDACTED`
